stdlib/websub/src/main/ballerina/websub/h2_hub_persistence.bal

```diff
@@ -65,7 +67,8 @@ public type H2HubPersistenceStore object {
         if (ret is sql:UpdateResult) {
             log:printDebug("Successfully removed " + ret.updatedRowCount + " entries for existing subscription");
         } else {
-            string errCause = <string> ret.detail().message;
+            error e = ret;
```

Comment:
Again why are we assigning to a new variable?

Reply:
In order to get the detail, type should be error. As per my above comment, with the recent error type change, updateResult inside type guard behaves as a union of errors(error|error).